stubs/jmespath/jmespath/exceptions.pyi

```diff
@@ -1,44 +1,41 @@
+from collections.abc import Sequence
 from typing import Any
 
 class JMESPathError(ValueError): ...
 
 class ParseError(JMESPathError):
-    lex_position: Any
-    token_value: Any
-    token_type: Any
-    msg: Any
-    expression: Any
-    def __init__(self, lex_position, token_value, token_type, msg=...) -> None: ...
+    lex_position: int
+    token_value: str
+    token_type: str
+    msg: str
+    expression: str | None
+    def __init__(self, lex_position: int, token_value: str, token_type: str, msg: str = ...) -> None: ...
 
 class IncompleteExpressionError(ParseError):
-    expression: Any
-    lex_position: Any
-    token_type: Any
-    token_value: Any
-    def set_expression(self, expression) -> None: ...
+    expression: str
```

Comment:
These are different to the superclass: https://github.com/jmespath/jmespath.py/blob/bbe7300c60056f52413603cf3e2bcd0b6afeda3d/jmespath/exceptions.py#L36-L37

```suggestion
    expression: str
    token_type: str | None
    token_value: str | None
```